src/Readline/HoaConsole.php

```diff
@@ -100,20 +100,10 @@ public function readline($prompt = null)
      * {@inheritdoc}
      */
     public function redisplay()
-    {
-        $this->_redisplay();
-    }
-
-    /**
-     * @return int
-     */
-    public function _redisplay()
     {
         $currentLine = $this->hoaReadline->getLine();
         Cursor::clear('all');
         echo $this->lastPrompt, $currentLine;
-
-        return HoaReadline::STATE_NO_ECHO;
```

Comment:
See ba7c3baef1dcb403716912a62bb26dddffe3f50a 342093a8aa3a9e8cad90053e6d88751f1fa6d8f3

Do not remove this return value.  If this is omitted, a line feed will be output.

Reply:
You don't return that value anyway in the original code?

Reply:
```
    public function redisplay()
    {
        $this->_redisplay();
    }
```

Reply:
Ah, I see the call. Fixed.